Hymns/Display/HymnLyricsView.swift

```diff
@@ -15,20 +15,23 @@ public struct HymnLyricsView: View {
         }
 
         return AnyView(
-            ScrollView {
-                VStack(alignment: .leading) {
-                    Group {
-                        ForEach(lyrics, id: \.self) { verse in
-                            Group {
-                                ForEach(verse.verseContent, id: \.self) { line in
-                                    Text(line)
+            VStack {
+                DetailHymnScreen()
+                ScrollView {
+                    VStack(alignment: .leading) {
+                        Group {
+                            ForEach(lyrics, id: \.self) { verse in
+                                Group {
+                                    ForEach(verse.verseContent, id: \.self) { line in
+                                        Text(line)
+                                    }
+                                    Spacer().frame(height: 30)
```

Comment:
This is fine for now but i might change it later. In my head, the lyrics view would go inside the detail view (called DisplayHymnView in Android), kind of like

DisplayHymnView {
  @State activeTab
  Tab(lyrics, chords, guitar piano)
   if activeTab == lyrics: show HymnLyricsView
   if activeTab == chords: show ChordsView
    ...
}